api/room/schema.py

```diff
@@ -88,6 +88,12 @@ def mutate(self, info, office_id, **kwargs):
 
         admin_roles.create_rooms_update_delete_office(office_id)
         query = Room.get_query(info)
+        query_result = query.filter(
+            RoomModel.calendar_id == kwargs.get('calendar_id'))
+        if query_result.count() > 0:
+            ErrorHandler.check_conflict(
+                self, kwargs['calendar_id'], 'CalenderId')
```

Comment:
do as @caveinn has suggested and all will be cool, your query should be something like 
```
admin_roles.create_rooms_update_delete_office(office_id)
query = Room.get_query(info)
active_rooms = query_room.filter(RoomModel.state == "active")
query_result = active_rooms .filter(
            RoomModel.calendar_id == kwargs.get('calendar_id'))
--------------------------------
*you other code here*
--------------------------------
```

Reply:
thanx @michael-basweti  this has been implemented.